### ISSUE TITLE
[BUG] Simple table foramt cannot be parsed properly

### ISSUE THREAD

**Issue by emisjerry** [ganesshkumar/obsidian-table-editor#38]
A simple Markdown table cannot be parsed.

```
Number | Name | Value
--|--|--
111|222|333
444|555|666
777|888|999
```

![image](https://user-images.githubusercontent.com/131511/167351944-94ecf46f-afda-4d05-a3d1-ce2ca4ee17eb.png)

Obsidian: v0.14.9
Plugin version: the newest one.

**Comment by ganesshkumar** [comment]
Yes, as of today, the plugin expects the input to be of the form 

```
|Number | Name | Value|
|--|--|--|
|111|222|333|
|444|555|666|
|777|888|999|
```

I wasn't aware of the fact that we can ignore the leading and trailing '|' from the table. Will fix this in the next iteration.
